models/lido/liquidity/zksync/lido_liquidity_zksync_maverick_pools.sql

```diff
@@ -7,7 +7,8 @@
     post_hook='{{ expose_spells(blockchains = \'["zksync"]\',
                                 spell_type = "project",
                                 spell_name = "lido_liquidity",
-                                contributors = \'["pipistrella"]\') }}'
+                                contributors = \'["pipistrella"]\') }}',
+    incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.time')]
```

Comment:
yep -- this should be good!
one minor request: can we move it up above the post hook? after the unique key?